### ISSUE TITLE
Make parameters metadata accessible through Python and the Web API

### ISSUE THREAD

**Issue by fpagnoux** [openfisca/openfisca-core#677]
In a YAML parameter file, I can define some metadata, for instance a `unit` and a `reference`:

```YAML
description: Taux d'imposition maximal
reference: "http://legifrance/..."
unit: /1
values:
  2006-01-01:
    value: 0.6
  2007-01-01:
    value: 0.5
  2011-01-01:
    value: null
```

But these metadata are not accessible neither in Python, nor through the Web API.

-----

As a client of the API building a front displaying the parameters,
I can access the metadata through the Web API
So that I can format my parameters accordingly

**Comment by benjello** [comment]
I am very interrested on getting the units via the python API. Any hint were I should start to

**Comment by Morendil** [comment]
Fixed by #681
